README.rst

```diff
@@ -163,6 +163,41 @@ Example usage:
         [pytest]
         postgresql_port = 8888
 
+Maintaining database state outside of the fixtures
+--------------------------------------------------
+
+It is possible and appears it's used in other libraries for tests,
+to maintain database state with the use of the ``pytest-postgresql`` internals
+but without postgresql fixtures.
+
+For this import DatabaseJanitor and use it's either init and drop methods:
```

Comment:
`and use its init and drop methods:`; `it is` is incorrect and `either` shouldn't be needed if the next example is in its own sentence.